akka-stream/src/main/scala/akka/stream/impl/io/TcpConnectionStream.scala

```diff
@@ -216,7 +217,14 @@ private[akka] abstract class TcpStreamActor(val settings: ActorFlowMaterializerS
 
   final override def receive = new ExposedPublisherReceive(activeReceive, unhandled) {
     override def receiveExposedPublisher(ep: ExposedPublisher): Unit = {
+      import context.dispatcher
       primaryOutputs.subreceive(ep)
+      subscriptionTimer = Some(
+        context.system.scheduler.scheduleOnce(
+          settings.subscriptionTimeoutSettings.timeout,
```

Comment:
Ah you mean, that does not use it, either. Anyway we pushed the whole issue to 1.x, so I will not touch it.